src/components/common/BuyCrypto.js

```diff
@@ -21,11 +20,9 @@ export default class BuyCrypto extends Component<Props> {
     return wallet.currencyNames[wallet.currencyCode]
   }
 
-  goToBuy = () => Actions[PLUGIN_LIST]({ direction: 'buy' })
-
   render () {
     return (
-      <TouchableWithoutFeedback onPress={this.goToBuy}>
+      <TouchableWithoutFeedback onPress={Actions.pluginBuy}>
```

Comment:
Use the Constants instead of hard coding the parameter. ie Actions[Constants.PLUGIN_LIST]

Reply:
fixed